python/ray/_private/ray_logging/formatters.py

```diff
@@ -45,10 +54,15 @@ def __init__(self) -> None:
         self._inner_formatter = logging.Formatter(LOGGER_FORMAT)
 
     def format(self, record: logging.LogRecord) -> str:
-        s = self._inner_formatter.format(record)
         record_format_attrs = generate_record_format_attrs(
             self, record, exclude_standard_attrs=True
         )
+
+        # Set the `created` field for the LogRecord so that the `asctime`
+        # generated by the `format` function is consistent with the `timestamp_ns`.
+        record.created = record_format_attrs[LogKey.TIMESTAMP_NS] / 1e9
```

Comment:
`record.created` is already set on line 19 above right, are there some other mechanisms to change the attributes again?

Reply:
good catch

Reply:
update https://github.com/ray-project/ray/pull/46074/commits/2504ab802c5ee0bfaac1f24b35317e36d32fa995